.github/workflows/continuous-integration.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - name: Check format
         run: yarn format:check
 
+      - name: Format code
```

Comment:
Please don't mix stuff from other tasks

Reply:
I had to put this because the prettier don't let me continue